README.md

```diff
@@ -15,7 +23,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Nix
-        uses: DeterminateSystems/nix-installer-action@v2
+        uses: DeterminateSystems/nix-installer-action@v3
```

Comment:
We should note in order for the flake checker functionality in the nix-installer-action to work, you need to check out your code before installing Nix.

Installing Nix, then checking out your code, wouldn't work. (Despite being a perfectly valid use case without this feature)

Reply:
Since this is an experiment and our example shows it checking out before installing, I think it is okay. However, I left a suggestion above :)